packages/opentelemetry-semantic-conventions/package.json

```diff
@@ -10,7 +10,7 @@
     "lint:fix": "eslint . --ext .ts --fix",
     "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
     "tdd": "npm run test -- --watch-extensions ts --watch",
-    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "codecov": "",
```

Comment:
I think we can remove it altogether since lerna will handle if its not present

Reply:
```suggestion
```